api-client/src/endpoints/createCitiesEndpoint.ts

```diff
@@ -36,7 +37,10 @@ export default (baseUrl: string): Endpoint<void, Array<CityModel>> =>
               prefix: city.prefix,
               longitude: city.longitude,
               latitude: city.latitude,
-              aliases: city.aliases
+              aliases: city.aliases,
+              boundingBox: Array.isArray(city.bounding_box)
+                ? (city.bounding_box.reduce((minPoint, maxPoint) => [...minPoint, ...maxPoint]) as BBox)
```

Comment:
if someone finds a good way to type it, feel free :)

Reply:
I think you mean [1] in the second case

Reply:
I do :grin:

Reply:
If you feel like the current solution is easier/more intuitive just keep it, just stumbled upon it. I also think you are working too late ^^

Reply:
no i thought omg why was your solution not my first idea...